Nodejs/Product/Nodejs/Snippets/1033/Nodejs/server.snippet

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>server</Title>
+      <Description>Code Snippet for a basic server</Description>
+      <Shortcut>server</Shortcut>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>message</ID>
+          <Default>message</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[var http = require('http');
+var server = http.createServer(function (request, response) {
+    response.writeHead(200, { "Content-Type": "text/plain" });
+    response.end("$message$");
+});
+server.listen(80);

```

Comment:
Instead of 80, use process.env.PORT || 3000 to be consistent with our other project templates.

process.env.PORT is also more likely correct and is going to work in deployment scenarios. When you deploy a node app, the PORT environment variable is typically set by the production server (which will forward that port as appropriate.) And in the event that PORT doesn't exist, you want to include a fallback port.